examples/tutorials/core_api_mnist/model_def.py

```diff
@@ -0,0 +1,147 @@
+# copy of https://github.com/pytorch/examples/blob/main/mnist/main.py renamed for consistency
```

Comment:
`# The starting script. This script is the same as [main.py](https://github.com/pytorch/examples/blob/main/mnist/main.py) without the Determined integrations.`